src/inboxsdk.d.ts

```diff
@@ -321,16 +321,40 @@ export class RouteView extends EventEmitter {
 export interface RowDescriptor {
   /** First textual column */
   title: string;
-  /** Second textual column */
+  /**
+   * @deprecated alias for {@link RowDescriptor#snippet}.
+   */
   body: string;
+  /** Second textual column. After {@link RowDescriptor#labels} if they're provided. */
+  snippet?: string | ((args: { el: HTMLElement; onDestroy(): void }) => void);
```

Comment:
Ok, we might move all of our onDestroy stuff (in this new API) over to this destroy => unmount.

```suggestion
  snippet?: string | ((args: { el: HTMLElement; unmountPromise: Promise<void> }) => void);
```

Reply:
[dd33e41](https://github.com/InboxSDK/InboxSDK/pull/1133/commits/dd33e415cdf3defb81b60da315fa8deb5d1a7979) includes a draft shape for how we'd handle arbitrary HTML.